### ISSUE TITLE
Error checking JSON files

### ISSUE THREAD

**Issue by TriMoon** [massivemadness/Squircle-CE#87]
**App Version:**
2021.1.3 (10006)

**Affected Device(s):**
Huawei P20 pro with Android 10.0

**Describe the bug**
When you have a (valid) JSON file opened in the editor and select the menu to check for errors you get an error "Unable to parse unsupported language".

**To Reproduce**
1. Check with below file: `Şarj yardımcısı.json`
```json
{
  "version": "1.0.0",
  "tts": {
    "engine": "default",
    "lang": "default"
  },
  "repeat": 15,
  "perc": 5,
  "reqPerc": 20,
  "fullPerc": 90,
  "tempUnit": 0,
  "battCapacity": 0,
  "chargeStop": {
    "capacity": 98,
    "temp": 0
  }
}
```

**Expected behavior**
No error shown or detected. 😸

**Comment by massivemadness** [comment]
Hi @TriMoon, it's because the editor doesn't have any json parser implementation. At this moment, only javascript is supported ☹️

**Comment by massivemadness** [comment]
Just released [v2023.1.0](https://github.com/massivemadness/Squircle-CE/releases/tag/2023.1.0) where I completely removed error checking. It'll be rewritten from scratch in the future due to large amount of problems
